src/views/HomeView.vue

```diff
@@ -172,4 +186,8 @@ h2 {
   width: 100px;
   margin-bottom: 20px;
 }
+
+.example-showcase .el-loading-mask {
```

Comment:
example-showcase? This el-loading-mask appears to be good in the components.scss